spec/git_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './git_clases.rb'
+
+RSpec.describe Gist do
+    describe "#post" do
+      it "crear gist" do
+        filename = "test.txt"
```

Comment:
These variables are shared amongst tests. What do you think about defining them just once at the very top?